src/libs/SearchUtils.ts

```diff
@@ -301,7 +304,31 @@ function isSearchResultsEmpty(searchResults: SearchResults) {
     return !Object.keys(searchResults?.data).some((key) => key.startsWith(ONYXKEYS.COLLECTION.TRANSACTION));
 }
 
+// This function should replace the current getQueryParams after the search v2.1 implementation.
```

Comment:
Let's remove this comment. I'll create a follow up issue to clean this up

Reply:
done